pcsx2/gui/ConsoleLogger.cpp

```diff
@@ -38,43 +38,44 @@ PipeRedirectionBase::~PipeRedirectionBase() = default;
 
 // ----------------------------------------------------------------------------
 //
-void pxLogConsole::DoLogRecord(wxLogLevel level, const wxString &message, const wxLogRecordInfo &info)
+void pxLogConsole::DoLogRecord(wxLogLevel level, const wxString & message, const wxLogRecordInfo & info)
 {
-	switch ( level )
+	switch (level)
 	{
-		case wxLOG_Trace:
```

Comment:
Switch cases still have wrong formatting.

Reply:
Yeah using the default clang format broke it, back to LLVM clang. Pushed